uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -297,8 +297,8 @@ define([
      * Sets the "change_balance" attribute to the leave request
      * if a force balance change recalculation is needed
      */
-    function decideIfBalanceChangeNeedsAForceRecalculation () {
-      if (!vm.isRole('staff')) {
+    function decideIfBalanceChangeNeedsRecalculationOnBackend () {
```

Comment:
`isBalanceChangeNeedsRecalculationOnBackend` ? Does it sound better(not sure)

Reply:
`isBalanceChangeRecalculationNeededOnBackend`